packages/carousel/src/Carousel.js

```diff
@@ -53,7 +70,34 @@ export const Carousel = ({
 
 Carousel.propTypes = {
   children: PropTypes.arrayOf(PropTypes.node).isRequired,
+  /** Number of slides to display at once. If using `layout`,
+   * this will be overridden by the number of items in the layout.
+   * E.g. `layout={'25-50-25'}` will result in 3 slides shown per page. */
   visibleSlides: PropTypes.number,
+  /** Show clickable navigation dots */
   showDots: PropTypes.bool,
+  /** Show forward/back navigation buttons */
   showForwardBackBtns: PropTypes.bool,
+  /** Percent width for each of the first N slides to use as a layout. Remaining slides will be divided
+   * into pages with the same number of items as the "layout" page. Not compatible with vertical orientation. */
+  layout: PropTypes.oneOf(['25-25-25-25', '33-33-33', '50-50', '40-60', '60-40', '25-75', '75-25']),
```

Comment:
Should it be `60-40` or do we want it to be `66-33`?

Reply:
I got these values from the Contentful model that Marketing will be using. @wesleymartin85 can you confirm these values are correct?